src/Orc.CsvTextEditor/Operations/AddColumnOperation.cs

```diff
@@ -7,6 +7,62 @@
 
 namespace Orc.CsvTextEditor.Operations
 {
+    internal class QuoteColumnOperation : OperationBase
+    {
+        #region Constructors
+        public QuoteColumnOperation(ICsvTextEditorInstance csvTextEditorInstance)
+            : base(csvTextEditorInstance)
+        {
+        }
+        #endregion
+
+
+        public override void Execute()
+        {
+            var location = CsvTextEditorInstance.GetLocation();
+            var startPosition = location.Column.Offset + location.Line.Offset;
+            var endPosition = startPosition + location.Column.Width;
+
+            var text = CsvTextEditorInstance.GetText();
+            text = TryRemoveQuoteFromPosition(endPosition - 2, text, out var quotesRemoved);
+            text = TryRemoveQuoteFromPosition(startPosition, text, out quotesRemoved);
+
+            var offsetDelta = -1;
+            if (!quotesRemoved)
+            {
+                var quoteStr = Symbols.Quote.ToString();
+                text = text.Insert(startPosition, quoteStr)
+                    .Insert(endPosition, quoteStr);
+
+                offsetDelta = 1;
+            }
+
+            CsvTextEditorInstance.SetText(text);
+
+            CsvTextEditorInstance.GotoPosition(location.Offset + offsetDelta);
+        }
+
+        private static string TryRemoveQuoteFromPosition(int symbolPosition, string text, out bool quotesRemoved)
```

Comment:
It feels the reverse. Normally in .NET, the Try[something] means returning a boolean (success or not) and the out parameter will be the updated value.

```
private static bool TryRemoveQuoteFromPosition(int symbolPosition, string text, out var resultText)
```

Reply:
yes...i know, i feel bad about it too).... but that time it was convenient... let me look what can i do with it